### DISCUSSION TITLE
Complete the expression ✍️

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#1276] [Galaxy Trivia] [answered]
> The greatest glory in living lies not in never falling, ...

**Bonus:** Name the author.

**Comment by tomerf** [comment]
"The greatest glory in living lies not in never falling, but in rising every time we fall." was said by Nelson Mandela.

**Comment by peterbaccaro** [comment, marked as answer]
**"The greatest glory in living lies not in never falling, but in rising every time we fall."** is a quote attributed to **Confucius**, the ancient Chinese philosopher.

**Reply by ParanoidUser** [reply]
Good job digging down to the roots! ⛏️👷‍♂️
